### PR TITLE
Definition wai aria specifications

### PR DESCRIPTION
**Pull request by Jym77**
Create a definition of "WAI-ARIA specifications" and use it.

Closes issue(s): 
- #751 

Need for Final Call:
This will require a 1 week Final Call (mostly grouping of definition but might have subtle implications in some rules)

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Comment by jkodu** [comment]
@Jym77 please resolve the conflicts

File: pages/glossary/explicit-role.md [hunk 1]
```diff
@@ -5,13 +5,7 @@ key: explicit-role
...
 The explicit semantic role is the [semantic role](#semantic-role) of an element set by its [role attribute](https://www.w3.org/TR/role-attribute/).
 
-The [role attribute](https://www.w3.org/TR/role-attribute/) takes a list of tokens. The explicit semantic role is the first valid role in this list. If none of the tokens are valid, the [implicit semantic role](#implicit-role) will be used instead.
-
-Non-abstract roles defined in the following specifications are considered valid:
-
-- [Accessible Rich Internet Applications (WAI-ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/)
-- [Digital Publishing WAI-ARIA Module 1.0](https://www.w3.org/TR/dpub-aria-1.0/)
-- [WAI-ARIA Graphics Module](https://www.w3.org/TR/graphics-aria-1.0/)
+The [role attribute](https://www.w3.org/TR/role-attribute/) takes a list of tokens. The explicit semantic role is the first valid role in this list. If none of the tokens are valid, the [implicit semantic role](#implicit-role) will be used instead. The valid roles are all non-abstract role from [WAI-ARIA Specifications](#wai-aria-specifications).
```
Comment (daniel-montalvo) [inline pages/glossary/explicit-role.md]:
```suggestion
The [role attribute](https://www.w3.org/TR/role-attribute/) takes a list of tokens. The explicit semantic role is the first valid role in this list. If none of the tokens are valid, the [implicit semantic role](#implicit-role) will be used instead. The valid roles are all non-abstract roles from [WAI-ARIA Specifications](#wai-aria-specifications).
```

**Comment by Jym77** [comment]
Final Call ends on October 17th.

**Comment by Jym77** [comment]
Final Call has ended. Merging.

**Pull request merged**